app/models/apartment.rb

```diff
@@ -0,0 +1,3 @@
+class Apartment < ApplicationRecord
+    belongs_to :user
```

Comment:
HEADS UP: Just be sure to remain consistent with the indentation (2 or 4 spaces), thank you!